README.md

```diff
@@ -3,7 +3,7 @@
 ![Swift5](https://img.shields.io/badge/swift-5-blue.svg)
 ![macOS](https://img.shields.io/badge/os-macOS-green.svg?style=flat)
 ![tuxOS](https://img.shields.io/badge/os-tuxOS-green.svg?style=flat)
-![Travis](https://travis-ci.org/NozeIO/swift-nio-redis-client.svg?branch=develop)
```

Comment:
Why disable Travis? Doesn't hurt to run that as well?

Reply:
This doesn't disable travis. It just replaces the build badge in the README. Do we want both badges? Just the github one? Just the travis one? I don't care.